src/illustrations/OverlapBox.jsx

```diff
@@ -1,19 +1,30 @@
-import React from "react";
 import PropTypes from "prop-types";
+import React from "react";
+import { withTranslation } from "react-i18next";
 import ReactTooltip from "react-tooltip";
-import Sleeve from "./Sleeve";
-import Outseam from "./Outseam";
-import Pinch from "./Pinch";
-import Shoe from "./Shoe";
-import FrontHeight from "./FrontHeight";
+import ProductModel from "../api/ProductModel";
 import Chest from "./Chest";
-import ShirtWaist from "./ShirtWaist";
-import PantWaist from "./PantWaist";
+import FrontHeight from "./FrontHeight";
 import Hips from "./Hips";
-import ShirtHips from "./ShirtHips";
+import Outseam from "./Outseam";
 import "./OverlapBox.scss";
-import ProductModel from "../api/ProductModel";
-import { withTranslation } from "react-i18next";
+import PantWaist from "./PantWaist";
+import Pinch from "./Pinch";
+import ShirtHips from "./ShirtHips";
+import ShirtWaist from "./ShirtWaist";
+import Shoe from "./Shoe";
+import Sleeve from "./Sleeve";
+
+const inchFractionOptions = {
+    0: "",
+    1: "⅛",
+    2: "¼",
+    3: "⅜",
+    4: "½",
+    5: "⅝",
+    6: "¾",
+    7: "⅞"
+};
```

Comment:
tää kans mieluummin arrayksi.. ja ehkäpä vielä jonnekin yhteiseen utility-moduliin, ettei näitä tartte kopsailla komponentista toiseen